lib/ecto/query/builder.ex

```diff
@@ -257,6 +257,16 @@ defmodule Ecto.Query.Builder do
     {{:{}, [], [:filter, [], [aggregate, filter_expr]]}, params_acc}
   end
 
+  def escape({:coalesce, _, values}, type, params_acc, vars, env) do
```

Comment:
Why do we have this clause?

Reply:
If it is for varargs, let's not add it now as not all of us agree with it.

Reply:
It was accidentally left in :/